src/arp/domain/ArpDomain.hx

```diff
@@ -83,7 +85,7 @@ class ArpDomain {
 
 		this._rawTick.push(this.onRawTick);
 
-		this.addTemplate(DataGroup);
+		this.addTemplate(DataGroup, true, OverwriteStrategy.Merge);
```

Comment:
This definitely should be defined by `@:arpType` metadata, but we'll never use `Merge` strategy for anything except `DataGroup` so we'll just merge and fix later :)